server/apps/immich/src/modules/immich-jwt/immich-jwt.service.spec.ts

```diff
@@ -1,57 +1,96 @@
 import { Logger } from '@nestjs/common';
 import { JwtService } from '@nestjs/jwt';
 import { Request } from 'express';
+import { UserEntity } from '../../../../../libs/database/src/entities/user.entity';
 import { LoginResponseDto } from '../../api-v1/auth/response-dto/login-response.dto';
+import { AuthType } from '../../constants/jwt.constant';
 import { ImmichJwtService } from './immich-jwt.service';
 
 describe('ImmichJwtService', () => {
-  let jwtService: JwtService;
-  let service: ImmichJwtService;
+  let jwtServiceMock: jest.Mocked<JwtService>;
+  let sut: ImmichJwtService;
 
   beforeEach(() => {
-    jwtService = new JwtService();
-    service = new ImmichJwtService(jwtService);
+    jwtServiceMock = {
+      sign: jest.fn(),
+      verifyAsync: jest.fn(),
+    } as unknown as jest.Mocked<JwtService>;
+
+    sut = new ImmichJwtService(jwtServiceMock);
   });
 
   afterEach(() => {
     jest.resetModules();
   });
 
+  describe('getCookieNames', () => {
+    it('should return the cookie names', async () => {
+      expect(sut.getCookieNames()).toEqual(['immich_access_token', 'immich_auth_type']);
+    });
+  });
+
   describe('getCookies', () => {
     it('should generate the cookie headers', async () => {
-      const spy = jest.spyOn(jwtService, 'sign');
-      spy.mockImplementation((value) => value as string);
+      jwtServiceMock.sign.mockImplementation((value) => value as string);
       const dto = { accessToken: 'test-user@immich.com', userId: 'test-user' };
-      const cookies = await service.getCookies(dto as LoginResponseDto);
+      const cookies = await sut.getCookies(dto as LoginResponseDto, AuthType.PASSWORD);
```

Comment:
This `await` shouldn't be necessary since `getCookies` returns an array

Reply:
Good catch.